### DISCUSSION TITLE
How to update the git clone branch label

### DISCUSSION THREAD

**Discussion by liverpoolfc-fan** [Z-Hub/Z-Push#14] [Q&A] [answered]
I cloned the repository a few days ago to run the latest 2.7.0 but when I run z-push-top.php it shows 
Z-Push:   develop-2.6.2-164-g6458636d
How do we change the version of the develop branch to reflect that 2.7.0 was released and changes are now being made to that baseline?

**Comment by matidau** [comment]
This comes from the tag, but the 2.7.0 didn't quite stick, I've just given it a 2.7.x for the time being until the next release.

It should produce:
develop-2.7.x
or something equivalent

**Comment by Danrancan** [comment]
> I cloned the repository a few days ago to run the latest 2.7.0 but when I run z-push-top.php it shows Z-Push: develop-2.6.2-164-g6458636d How do we change the version of the develop branch to reflect that 2.7.0 was released and changes are now being made to that baseline?

In Linux try going into the source folder and running:
`git checkout release/2.7`
That should do the trick.

**Reply by matidau** [reply, marked as answer]
We are using Githubs Releases for the tagging. Updating the Tag in develop resolved this. The 2.7.x will get dropped after the next release.
